src/components/Drawer/index.js

```diff
@@ -57,22 +51,14 @@ export default function Drawer(props) {
     );
 
     useEffect(() => {
-        const contentElement = contentRef.current;
```

Comment:
seems that contentRef is not being used anymore then let's clean all code related to it